nuxeo-services/nuxeo-platform-convert/src/test/java/org/nuxeo/ecm/platform/convert/tests/TestAnyToPDFConverters.java

```diff
@@ -184,5 +123,4 @@ public void testMultiThreadsConverter() throws Exception {
         assertFalse(t1.exception);
         assertFalse(t2.exception);
     }
-
```

Comment:
Don't remove empty lines (formatting) in parts of the code that aren't touched by the patch.